### DISCUSSION TITLE
how should i make login system using uppaal really dont understand

### DISCUSSION THREAD

**Discussion by bibikazmi4** [UPPAALModelChecker/UPPAAL-Meta#260] [General]
how should i make login system using uppaal really dont understand kindly pls someone tell me how the diagram will look like if someone show me the diagram i understand quickly someone pls

**Comment by mikucionisaau** [comment]
Do you mean how to make a model of a login system?
Is that your homework assignment?
It's quite difficult to guess the actual problem (I guess that's the whole point of the exercise), but try to think in terms of communicating processes: one template per process, add locations for various stages, add edges with synchronizations in between locations.
Be smart: make an attempt yourself first and when stuck, ask your instructor first.

**Comment by bibikazmi4** [comment]
Yeah its my assignment i can't ask my instructor i am stuck right now so
please help me
